receiver/mysqlreceiver/metadata.yaml

```diff
@@ -279,3 +283,13 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.mysqlx_connections:
+    enabled: true
+    description: The number of mysqlx connections.
```

Comment:
This metric appears to be specific to the "X Plugin". I think we need to call out when it will be present and provide a link for context. Probably use the `extended_documentation` field for this.

Reply:
Additional documentation reference added.

I would leave it as default, as there is no additional cost related to that (except traffic). WDYT?

Reply:
👍 We're not making any additional requests to get these data points. We're just rendering the metric if the corresponding keys are present in the result. Might as well return them if they are there.